src/ListGroup.js

```diff
@@ -35,6 +35,10 @@ class ListGroup extends React.Component {
     }
   }
 
+  isAnchor(props){
+    return (props.href || props.target || props.onClick);
```

Comment:
If that's the case, should the exclusion apply to `ListGroupItem.js` as well? 
Line 34 checks for target to render anchor tags.